File: Vitoria-e-Sofia/CPP/Lista-1/ex-03.cpp
/*
Data: 05/05/2026
Autores: Vitória Kishimoto e Sofia Franzini
Descrição:Ler vários números até que seja digitado um número negativo. Calcular e exibir a soma desses números.
*/

#include <iostream>
using namespace std;

int main() {
    double numero, soma = 0;

    cout << "Digite números para somar." << endl;
    cout << "Digite um número negativo para encerrar." << endl;

    while (true) {
        cin >> numero;

        if (numero < 0) {
            break;
        }

        soma += numero;
    }

    cout << "Soma total: " << soma << endl;

    return 0;
}

File: Vitoria-e-Sofia/Python/python-conversion/python/ex03.py
"""
    Autores: Vitória Kishimoto e Sofia Franzini
    Data: 10/08/26
    Descrição: Lê um número inteiro positivo e exibe a contagem de 1 até esse número.
"""

numero= int(input("Digite um número inteiro positivo: "))
contador= 1

if numero > 0 :
    while contador <= numero:
        print (contador)
        contador += 1

else : 
    print ("Número inválido. Digite um valor positivo")
    
